ggml.h

```diff
@@ -1599,8 +1600,8 @@ extern "C" {
             int                   k1,
             int                   s0,
             int                   s1,
-            float                 p0,
-            float                 p1);
+            int                   p0,
+            int                   p1);
```

Comment:
I think there was a reason for these to be float. @ggerganov

Reply:
Yes, the `yolo` example relies on this:

https://github.com/ggerganov/ggml/blob/475cbad5c1c834e31e26a2283bc1413181644360/examples/yolo/yolov3-tiny.cpp#L398-L400

Context: https://github.com/ggerganov/ggml/pull/576#issue-1939536851

Reply:
fallback in 66dd123b0f54bd1a6baaaa1c25c322c0132d2312